ptu/global.ini

```diff
@@ -4206,15 +4206,15 @@ DestroyObj_Text_Long=Begib dich zum Port Olisar-Station und triff dich dort.
 DestroyObj_Text_Marker=Treffen
 DestroyObj_Text_Short=Treffen bei Olisar
 Dlg_SC_Tutorial_2nd_half_introTraining_1050_2nd_Half_Intro=Aufgetankt. Lass uns wieder rausgehen.
-Dlg_SC_Tutorial_2nd_half_launchTraining_1060_2nd_Half_Launch=Heb ab, wenn du bereit bist.
+Dlg_SC_Tutorial_2nd_half_launchTraining_1060_2nd_Half_Launch=Hebe ab, wenn du bereit bist.
 Dlg_SC_Tutorial_adjust_backTraining_0250_Take_Off_Player_Adjust_Back=Etwas zurück.
 Dlg_SC_Tutorial_adjust_forwardTraining_0240_Take_Off_Player_Adjust_Forward=Etwas vorwärts.
 Dlg_SC_Tutorial_adjust_leftTraining_0260_Take_Off_Player_Adjust_Left=Ein wenig nach links.
-Dlg_SC_Tutorial_adjust_rightTraining_0270_Take_Off_Player_Adjust_Right=Nach rechts ein bisschen.
-Dlg_SC_Tutorial_approach_landing_padTraining_0850_Landing_Gentle_Encouragement=Setz sie sanft ab.
+Dlg_SC_Tutorial_adjust_rightTraining_0270_Take_Off_Player_Adjust_Right=Ein wenig nach rechts.
+Dlg_SC_Tutorial_approach_landing_padTraining_0850_Landing_Gentle_Encouragement=Setze sanft ab.
```

Comment:
Hier ist landen gemeint. Also "Setze sanft auf".

Reply:
Ändere ich in einem anderen PR